Gemfile

```diff
@@ -20,7 +20,7 @@ else
   gem 'rake', '< 11.0.0' # rake 11 requires Ruby 1.9.3 or later
 end
 
-gem 'yard', '~> 0.9.12', :require => false
+gem 'yard', '~> 0.9.22', :require => false
```

Comment:
Can we limit this to 2.7.0 only until we resolve the docs differences? Unless you want to go through the diff now?